rotkehlchen/db/upgrades/v39_v40.py

```diff
@@ -245,6 +249,45 @@ def _migrate_ledger_actions(write_cursor: 'DBCursor', conn: 'DBConnection') -> N
     log.debug('Exit _migrate_ledger_actions')
 
 
+def _migrate_ledger_airdrop_accounting_setting(write_cursor: 'DBCursor') -> None:
+    """
+    Migrates the accounting setting for airdrops to the new table for accounting
+    rules. It requires the existence of the accounting_rules table and the existence of
+    `taxable_ledger_actions` on the settings table.
+    """
+    # copy the taxable ledger actions rules
+    new_accounting_rule = None
+    write_cursor.execute('SELECT value FROM settings WHERE name=?', ('taxable_ledger_actions',))
+    if (taxable_ledgers_row := write_cursor.fetchone()) is not None:
+        try:
+            taxable_ledger_events = json.loads(taxable_ledgers_row[0])
+        except json.JSONDecodeError as e:
+            log.error(f'Failed to decode ledger action taxable rules {e}')
+        else:
+            if 'airdrop' in taxable_ledger_events:
+                new_accounting_rule = (
+                    HistoryEventType.RECEIVE.serialize(),
+                    HistoryEventSubType.AIRDROP.serialize(),
+                    NO_ACCOUNTING_COUNTERPARTY,
+                    *TxEventSettings(
+                        taxable=True,  # make it taxable
```

Comment:
nitpick: obvious comment is obvious

```suggestion
                        taxable=True, 
```